tests/karate/src/test/java/be/ServerTest.java

```diff
@@ -24,4 +24,21 @@ Karate testOpenRollCall() {
   Karate testCloseRollCall() {
     return Karate.run("classpath:be/createRollCall/closeRollCall.feature");
   }
+
+  @Karate.Test
+  Karate testElectionSetup(){
+    return Karate.run("classpath:be/createElection/electionSetup.feature");
+  }
+
+  @Karate.Test
+  Karate testCastVote(){
+    return Karate.run("classpath:be/createElection/castVote.feature");
```

Comment:
This is something I forgot to mention already for the Roll-call, but I think the folders should be called LAO, RollCall and election rather than add the creation